docs/octopus-rest-api/octopus.server.exe-command-line/configure.md

```diff
@@ -69,9 +69,14 @@ Where [<options>] is any of:
                              Comma-separated whitelist of domains that are
                                allowed to retrieve data (empty turns CORS off,
                                * allows all).
+      --xFrameOptions=VALUE  A directive to provide in the X-Frame-Options
```

Comment:
Not sure why this has disappeared. Having trouble keeping track of the [reverts of the reverts](https://github.com/OctopusDeploy/OctopusDeploy/commit/df27c9975bd43ab4007db2d7fdbec2411f82acc7).
Maybe @alialmaktoum can provide some insight into why it's not `2020.1.12`?